tutorsentry/patches/openedx-dev-dockerfile-post-python-requirements

```diff
@@ -0,0 +1,2 @@
+RUN --mount=type=cache,target=/openedx/.cache/pip,sharing=shared \
+    pip install "sentry-sdk==2.8.0"
```

Comment:
Why not `openedx-dockerfile-post-python-requirements` and avoid two patches?

Reply:
For easier testing of the developments on this plugin

Reply:
I'm mean, doesn't the `openedx-dockerfile-post-python-requirements` guarantees that the plugin is installed in both layers?

Reply:
Yes, it does